src/s3b_the_roach_market.tw2

```diff
@@ -0,0 +1,278 @@
+
+I was already past him, and waved my hand dismissively without looking back. “Yeah sure Dan, sorry or whatever. I’ll see ya around.”
+
+> [[Head to Auntie’s gift shop|RM-giftshop-entrance]]
+> [[Head to the junk heap|RM-junkheap-entrance]]
+
+::RM-DopemanDan-free-drugs
+Dan cackled. “Hell yeah! Now you’re talking words Dopeman understands.” He thrust the bag forward, and I took it.
+
+I unrolled the top and peeked inside. “Wow. This is, uh, a pretty full bag.”
+
+“Yeah, dawg. I figure she was either getting herself set up for a sweet-ass month or one fuckin’ buckwild night.”
+
+I was about to ask Dan if this was a regular thing for Anna, but then I thought better of it. I didn’t want him to think he had anything to say about her that I didn’t already know.
+
+“Well, cool, man. With a haul like this, I bet she’d never notice if a little went missing.” I winked at him.
+
+Dan tapped the side of his nose with his finger. “Trip in peace, my ‘lette. Dopeman ain’t never snitched on nobody.”
+
+I nodded, and made my way deeper into the Roach Market.
+
+> [[Head to Auntie’s gift shop|RM-giftshop-entrance]]
+> [[Head to the junk heap|RM-junkheap-entrance]]
+
+::RM-giftshop-entrance-nameprompt
+{ (set: $playerName to (prompt: "Enter name:", ""))
+(if: (text: $playerName)'s length > 1)[Right, my name is $playerName. [[Moving on|RM-giftshop-entrance]].<!-- what this code does is check the ending of the custom $playerName string to hopefully pluralize it properly -->{
+  (if: $playerName's last is "s")[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "s"))[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else-if: ($playerName's last is "h") and ($playerName's 2ndlast is "c"))[
+    (set: $playerNames to ($playerName + "es"))
+  ]
+  (else:)[
+    (set: $playerNames to ($playerName + "s"))
+  ]
+}]
+}(else:)[No, [[that's not right|RM-giftshop-entrance-nameprompt]].]
+
+::RM-giftshop-entrance
+{(set: $spokeToToe to true) <!-- forcing this to true for initial testing -->
+(if: $giftshopVisited is true)[
+<!-- alternate intro if revisiting the gift shop scene -->I was back in Auntie’s: her combination gift shop, living room, and bedroom, with cheap wire racks all around. Auntie was plopped in the middle of it all just like before, watching TV from her recliner. I think <i>Big Money Rustlas</i> was on. She loved the oldies.
+]
+(else:)[
+<!-- gift shop intro on first visit -->
+  (set: $giftshopVisited to true)
+Auntie’s “gift shop” was also her living room. And bedroom, I guess, since she usually fell asleep in the threadbare recliner she had set up in a clearing in the middle of everything. She kept the merchandise in cheap wire shelves standing in neat rows all around her, and she kept herself plopped in a chair watching a flickery TV. She had an end table on her right side, where a lamp with a yellowing shade sat on an honest-to-god white doily.
```

Comment:
This is another passage that feels dense because it drops all at once and it gets denser as the player reveals text for each object (I love the reveals of the indented descriptions).

Maybe we could have the description of Auntie's Shop in one passage and the indented reveals in a subsequent one? Or, and sorry to sound like a broken record, at least split up the shop description text with `click:` reveals so the player isn't seeing so much text at the start of the passage?

Reply:
Breaking this up more is needed too. Was hoping that other eyes might be able to help drill down into the pesky loading-speed bug first. Or is that not happening on this initial passage for others, when y'all play through it? `Looking around the racks, I saw three good possibilities to help me win Toe over:` loads in way slower when I play through

Reply:
Oh, I'm so sorry! I saw it in your commit description and when I was play testing, but I forgot to mention in my review. I honestly have no idea what's causing it. Any thoughts @whymog ?

Reply:
Weird. Not sure why this is the case, but when I replace the `<p>` tags on lines 113, 117, 123, and 125 with `<br><br>`, the staggered/inconsistent display issue went away. Can you reproduce this on your end, @thenthdegree? If so, we should do a full sweep of this file for stray, unclosed `<p>` tags and replace them with `<br>`s now to avoid any weirdness.